### DISCUSSION TITLE
v1.14 Release Candidate 1

### DISCUSSION THREAD

**Discussion by dorkster** [flareteam/flare-game#919] [Announcements]
## Engine features

* Added 'loot.requires_status' and 'status_loot' properties to loot table definitions.
* Support '/' as shorthand for 'exec' in the dev console. A space is not required after the slash.
* Added 'spawn_level' property to map enemies. This follows the same syntax as the Power property of the same name.
* Add XP scaling tables for scaling enemy reward XP based on enemy level.
* Replace 'immunity' effects with percentage-based resistances.
* Added 'mouse_move_deadzone' property to engine/misc.txt to help alleviate erratic mouse-move behavior when the cursor is near the player.
* Alternative tab UI for when tabs exceed the available space.
* Added keybind for cycling equipment sets in reverse.
* Refactored stats and stat bonuses to use floats for fractional values.
* Support fractional values in the following:
  * Combat text: 'speed'
  * Engine (combat): 'absorb_percent', 'resist_percent', 'block_percent', 'avoidance_percent', 'miss_damage_percent', 'crit_damage_percent', 'overhit_damage_percent'
  * Engine (misc): 'party_exp_percentage'
  * Events: 'power_damage', 'chance_exec'
  * Death penalty: 'currency', 'xp_total', 'xp_current_level'
  * Item: 'dmg', 'abs', 'bonus'
  * Map: 'enemygroup.chance'
  * Power: 'requires_mp', 'requires_hp', 'requires_hpmp_state', 'attack_speed', 'hp_steal', 'mp_steal', 'missile_angle', 'angle_variance', 'post_effect', 'post_effect_src', 'pre_power', 'modifier_accuracy', 'modifier_damage', 'modifier_critical'
  * StatBlock: 'stat', 'stat_per_level', 'stat_per_primary', 'vulnerable', 'chance_pursue', 'chance_flee'
* Added engine/number_format.txt. This file defines the number of digits to print after the decimal point for fractional values under different contexts.
* Add support for multiplicative stat bonuses. For example, an item bonus of 1.5x Max HP, which would be defined as 'bonus=hp,150%'.
  * As such, the 'hp_percent' and 'mp_percent' stat bonuses are considered deprecated.
* Players are directed to the Mods configuration menu via a dialog box when no core mod is enabled. (kr4ft3r)
  * Core mods are now labeled as such in their tooltips on the Mods configuration menu.

## Engine fixes

* Fixed set item tooltips showing incorrect number of equipped items if a set item was on an inactive loadout.
* Fixed dev console commands not working if they were preceded by whitespace.
* Fixed spawning entities not counting towards summoned entity count.
* Fix broken gamepad navigation after closing Powers menu while on a non-default tab.
* Removed hard-coded minimum values for HP/MP steal and damage return.
* Fixed "activate" message incorrectly showing in MenuPowers tooltips.
* Fix some cases where 'min' damage/absorb could be greater than the cooresponding 'max' value.
* Fixed text input issues with deleting text on Android.

## Game updates

* Fireball and Ice Bolt scrolls now give the player a 10 second period in which their basic attacks (Swing, Shoot, Channel) are replaced with the scroll power.
* Fixed the naming of scroll damage as it appears in power description tooltips.
* Lowered Lezaith's slow resistance to 50%, down from 100%
* Lowered Lezaith's stun resistance to 90%, down from 100%

## Translation updates

* German (de) update (Wuzzy2)
* Scottish Gaelic (gd) update (GunChleoc)
* Indonesian (id) update (Kirim Saja)
* Italian (it) update (Riccardi Unix)
* Japanese (ja) update (Sato Hayato)
* Korean (ko) update (ks k, YooJin919)
* Polish (pl) update (Pawonashi)
* Chinese (zh) update (Issac Zhang)

<hr /><em>This discussion was created from the release <a href='https://github.com/flareteam/flare-game/releases/tag/v1.14rc1'>v1.14 Release Candidate 1</a>.</em>

**Comment by WithinAmnesia** [comment]
Thank you @dorkster I really appreciate all the hard work you and everyone has put into making this happen. Flare is looking to be shaping up to be an epic RPG game engine that's open for everyone to use and make great games worth playing creatively empowering hobbyists and game studios living on a dream. Flare is looking to be shaping up to make great games and to inspire today's and tomorrow's generations of game developers. You are great Justin and you make everyone proud to work on such a great game engine! It sure has come a long way and its amazing the world has people like yourself and this super supportive community. I feel with one more push we can make a truly incredible epic game engine that can go toe to toe with the industry leading RPGs and win as individual game developers! Flare is so amazing and I hope it gets better and better. 1.14 looks great and I hope in 2023 we get 1.15 too and mate then Flare will be truly a super RPG engine that is incredibly epic. It's already epic but hopefully it can be even better with arbitrary definition of resources for an example a whole new world of possibilities with Heath, Spirit and Mana! Flare Is Empowering!

**Comment by WithinAmnesia** [comment]
May you please add https://github.com/flareteam/flare-engine/issues/1776 as the last major thing I could ask of you Justin? This is the final feature I need to make my dream RPG video game a reality. I sent you and email with more details.

**Comment by Blueberryy** [comment]
Where can i see roadmap? Like i hope there's some Trello board

**Reply by dorkster** [reply]
Nothing fancy. We just use Github milestones:

- Engine: https://github.com/flareteam/flare-engine/milestones
- Game: https://github.com/flareteam/flare-game/milestones

**Reply by Blueberryy** [reply]
Thanks
